math/TestTrigonometric.cpp

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2020 Nicholas Corgan
+// SPDX-License-Identifier: BSL-1.0
+
+#include <Pothos/Framework.hpp>
+#include <Pothos/Testing.hpp>
+
+#include <algorithm>
+#include <cstring>
+#include <iostream>
+#include <vector>
+
+#define _USE_MATH_DEFINES
```

Comment:
Pothos/Config.hpp actually sets this so we dont have to worry about sprinkling _USE_MATH_DEFINES around for windows

Reply:
Fixed.